File: frontend/front/src/app/model/coin.ts
export class Coin {
    name: string;
    price: number;
    gold: number;
    silver: number;
    bronze: number;

    constructor(name: string, price: number, gold: number, silver: number, bronze: number) {
        this.name = name;
        this.price = price;
        this.gold = gold;
        this.silver = silver;
        this.bronze = bronze;
    }
}

File: back/model/coin.py
from sqlalchemy import Column, Integer, String, Float
from sqlalchemy.orm import relationship

from database.database import Base, engine


class Coin(Base):
    __tablename__ = "coin"

    id = Column(Integer, primary_key=True, index=True)
    name = Column(String, index=True)
    price = Column(Float, nullable=False)
    gold = Column(Float, nullable=False)
    silver = Column(Float, nullable=False)
    bronze = Column(Float, nullable=False)

    orders = relationship("Order", back_populates="coin")
    trade_lots = relationship("TradeLot", back_populates="coin")

Coin.metadata.create_all(bind=engine)